src/Network/Request.php

```diff
@@ -135,17 +144,25 @@ protected function parseOrFail(array $payload): static {
 		static::validateInputFields($payload, static::PAYLOAD_FIELDS);
 
 		// Checking if request format and endpoint are supported
-		[$path] = explode('?', ltrim($payload['message']['path_query'], '/'));
-		$endpoint = match ($path) {
-			'cli' => ManticoreEndpoint::Cli,
-			'cli_json' => ManticoreEndpoint::CliJson,
-			'sql?mode=raw', 'sql', '' => ManticoreEndpoint::Sql,
-			'insert', 'replace' => ManticoreEndpoint::Insert,
-			'bulk' => ManticoreEndpoint::Bulk,
-			default => throw new InvalidRequestError(
-				"Do not know how to handle '{$payload['message']['path_query']}' path_query"
-			),
-		};
+		[$this->path] = explode('?', ltrim($payload['message']['path_query'], '/'));
+		if (preg_match('/\/\_doc(\/|$)/', $this->path)
+			|| preg_match('/\/\_create(\/|$)/', $this->path)) {
+			// We don't differentiate elastic-like insert and replace queries here
+			// since this is irrelevant for Buddy processing logic
+			$endpointBundle = ManticoreEndpoint::Insert;
+		} else {
+			$endpointBundle = match ($this->path) {
```

Comment:
I've commented about the change of the endpoint's type above.  As for `endpointBundle`, I think it sounds more logical since one 'bundle' can now comprise multiple actual endpoints(query pathes). E.g.,  `ManticoreEndpoint::Bulk`  should include both `bulk` and `_bulk` endpoints, etc.